src/docfx/lib/HtmlUtility.cs

```diff
@@ -406,8 +406,9 @@ internal static void RemoveRerunCodepenIframes(ref HtmlToken token)
 
         private static string AddLocaleIfMissingForAbsolutePath(string href, string locale)
         {
-            // should not add locale for api links
-            if (href.StartsWith("/api/", StringComparison.OrdinalIgnoreCase))
+            // should not add locale for api, _themes links
+            if (href.StartsWith("/api/", StringComparison.OrdinalIgnoreCase) ||
+                href.StartsWith("/_themes/", StringComparison.OrdinalIgnoreCase))
```

Comment:
I'd say for any routes starting with `/_` are private routes that we shouldn't be messing up with
```suggestion
                href.StartsWith("/_/", StringComparison.OrdinalIgnoreCase))
```